assets/header.css

```diff
@@ -250,7 +250,7 @@
   min-width: 230px;
   width: 100%;
   padding: 18px 18px 18px 67px;
```

Comment:
```suggestion
  padding: 18px;
  padding-left: 67px;
```

Reply:
Could you take at the same code snippets in the whole theme? As it's an improvement of the theme, IMO it's better to write 2 lines of code instead of more complex 4-sides declaration, especially when only top/bottom is set